test/cql/query_bounds_test.result

```diff
@@ -459,15 +459,21 @@ SELECT * FROM foo WHERE a=0 AND SCYLLA_CLUSTERING_BOUND(b) > SCYLLA_CLUSTERING_B
 -- too many values --
 SELECT * FROM foo WHERE a=0 AND (b, c) > SCYLLA_CLUSTERING_BOUND (1, 0, 5);
 {
-	"message" : "exceptions::invalid_request_exception (Expected 2 elements in value tuple, but got 3: (1, 0, 5))",
+	"message" : "exceptions::invalid_request_exception (Invalid tuple literal for b: too many elements. Type frozen<tuple<int, int>> expects 2 but got 3)",
 	"status" : "error"
 }
 
 -- too few values --
 SELECT * FROM foo WHERE a=0 AND (b, c) > SCYLLA_CLUSTERING_BOUND (1);
 {
-	"message" : "exceptions::invalid_request_exception (Expected 2 elements in value tuple, but got 1: (1))",
-	"status" : "error"
+	"rows" : 
+	[
+		{
+			"a" : "0",
+			"b" : "0",
+			"c" : "0"
+		}
+	]
```

Comment:
What's going on here? The test is called "too few values", and used to geenerate an error, but now suddenly doesn't generate an error. Why? Is it still "too few values"?

Reply:
Sending tuple with two values in place where tuple with three values is expected is sometimes correct.
For example if we send two `ints` to a `bind_variable` that has type `tuple<int, int, int>` Scylla will accept it and the third value will be `null`.

This is why validation code accepted this query - `tuple<int, int>` in place of `tuple<int, int, int>` is okay. I added checks for this specific case and now the test passes without any modifications.